Dockerfile

```diff
@@ -15,7 +15,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-ARG GO_VERSION=1.21.12
+ARG GO_VERSION=1.22.7
 ARG XX_VERSION=1.2.1
 ARG GOLANGCI_LINT_VERSION=v1.55.2
```

Comment:
Oh! You probably want to update GOLANGCI_LINT_VERSION as well for compatibility with newer go versions (CI still seemed happy, so maybe it still works); I think 1.60.2 is the current version, but you may run into linters complaining, so if it works, you could do it separately; see https://github.com/moby/moby/pull/48359

Reply:
The `poll.Continue(r.Combined())` ones will be addressed by https://github.com/gotestyourself/gotest.tools/pull/279, but in the meantime you can use;

```go
poll.Continue("%v", r.Combined())
```